collie/model/base/base_pipeline.py

```diff
@@ -664,8 +664,8 @@ def get_item_predictions(self,
         if user_id > self.hparams.num_users:
             raise ValueError(
                 f'``user_id`` {user_id} is not in the model. '
-                'Expected ID between ``0`` and ``self.hparams.num_users``, '
-                f'{self.hparams.num_users}'
+                'Expected ID between ``0`` and ``self.hparams.num_users`` '
+                f'({self.hparams.num_users}), not {user_id}'
```

Comment:
How do you feel about excessive back-ticks? 

```suggestion
                f'(``{self.hparams.num_users}``), not ``{user_id}``'
```